packages/sdk-core/default.nix

```diff
@@ -0,0 +1,29 @@
+{
+  halfBoardModule = {
+    dependencies = [
+      ../..
+      ../ethereum-contracts
+    ];
+    outputs = [
+      "dist"
+      "coverage"
```

Comment:
There might be more outputs? https://github.com/superfluid-finance/protocol-monorepo/blob/dev/packages/sdk-core/.gitignore